smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/commit/DeprecatedCommitStrategiesTest.java

```diff
@@ -323,6 +326,7 @@ void testThrottledStrategyWithTooManyUnackedMessages() throws Exception {
 
         consumer.schedulePollTask(() -> {
             consumer.rebalance(offsets.keySet());
+            source.getCommitHandler().partitionsAssigned(offsets.keySet());
```

Comment:
I'd expect identical changes in the `CommitStrategiesTest` -- this one is a copy of that using `@Named` instead of `@Identifier`.

Reply:
Fixed.

FYI, the mock consumer does not handle rebalance (does not call the listener) but we need to call rebalance to set up the assigned partitions. So adding this second invocation simulates a "real" rebalance.